starter_code/app.js

```diff
@@ -1,21 +1,35 @@
-
-const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
-const path    = require('path');
-const PunkAPIWrapper = require('punkapi-javascript-wrapper');
+const express = require("express");
+const hbs = require("hbs");
+const app = express();
+const path = require("path");
+const PunkAPIWrapper = require("punkapi-javascript-wrapper");
 const punkAPI = new PunkAPIWrapper();
-
-app.set('view engine', 'hbs');
-app.set('views', __dirname + '/views');
-app.use(express.static(path.join(__dirname, 'public')));
-
-
-
-app.get('/', (req, res, next) => {
-  res.render('index');
+app.set("view engine", "hbs");
+app.set("views", __dirname + "/views");
+app.use(express.static(path.join(__dirname, "public")));
+hbs.registerPartials(__dirname + "/views/partials");
+app.get("/", (req, res, next) => {
+  res.render("index");
+});
+app.get("/beer", (req, res, next) => {
+  punkAPI
+    .getBeers()
+    .then(beers => {
+      res.render("beer", { beers });
+    })
+    .catch(error => {
+      console.log(error);
+    });
+});
+app.get("/random-beers", (req, res, next) => {
+  punkAPI
+    .getRandom()
+    .then(beers => {
+      let beerss = beers[0];
+      res.render("random-beers", { beerss });
+    })
+    .catch(error => {
+      console.log(error);
+    });
 });
-
-
-
 app.listen(3000);
```

Comment:
Use a callback inside the app.listen with a console log in order to let you know when a server is running 😉 

``` 
app.listen(PORT, ()=>{
  console.log(`listen on port: ${PORT}`)
});